sky/clouds/service_catalog/data_fetchers/fetch_gcp.py

```diff
@@ -22,7 +22,8 @@
 
 # Supported GPU types and counts.
 GPU_TYPES_TO_COUNTS = {
-    'A100': [1, 2, 4, 8, 16],
+    'A100 40GB': [1, 2, 4, 8, 16],
+    'A100 80GB': [1, 2, 4, 8],
```

Comment:
Do we want to replace the space with `-`? We are using `V100-32GB` for example.

Reply:
Oh, just saw below. Best to add a comment here saying renamed later.

Reply:
@concretevitamin That's a great point. The two are official names used in the GCP pricing & region-zone tables. After crawling, we rename them as 'A100' and 'A100-80GB' for consistency. Added a comment on this. PTAL.